data_retriever/migration_event_queue.py

```
@@ -0,0 +1,204 @@
+            self._migration_id = ""
+        except Exception as e:
+            raise EventQueueException(f"Failed to connect to Postgres: {e}") from e
+
+    def disconnect(self):
+        """
+        Disconnect from Postgres
+        Raises:
+            EventQueueException: If deconnection could not be performed
+        """
+        if not self._conn or not self._cursor:
+            raise EventQueueException(f"Postgres connection not established")
```

Ruff (0.12.2)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)